css/style.css

```diff
@@ -48,22 +48,110 @@ span {
   display: inline-block;
 }
 
+/* stream player styling */
+#stream {
+  text-align: center;
+}
+
+/* Logo centering and scaling */
+#WMTU {
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  -webkit-flex-direction: row;
+  justify-content: center;
+  flex-wrap: nowrap;
+  -webkit-flex-wrap: nowrap;
+  margin-left: auto;
+  margin-right: auto;
+}
 
 /* logo styling */
-#top-logo {
-  width: 100%;
-  height: 300px;
+@media all and (max-width: 1499px) {
+  #top-logo {
+    width: auto;
+    max-height: 18em;
+  }
+}
+
+@media all and (min-width: 1500px) {
+  #top-logo {
+    width: auto;
+    height: 25em;
+  }
 }
 
+#logo {
+  order: 1;
+}
 
-/* stream player styling */
-#stream {
-  text-align: center;
+@media all and (min-width: 1500px) {
+  #player {
+    animation: pulse 2s infinite;
+    padding-right: 10px;
+    font-size: 7em;
+    order: 2;
+    position: relative;
+    top: 1.7em;
+    vertical-align: middle;
+    display: inline-block;
+    margin-left: -0.6em;
+    color: #000;
+  }
 }
 
-#player {
-  font-size: 5em;
-  color: #000000;
+@media all and (min-width: 476px) and (max-width: 1499px) {
```

Comment:
This is more of an implementation note, not something that absolutely has to be fixed: you can cascade media queries instead of defining strict ranges, which would allow you not have to repeat the same bits of styling for each range.